src/games/strategy/engine/framework/headlessGameServer/HeadlessGameServer.java

```diff
@@ -62,8 +63,7 @@
   private final ScheduledExecutorService m_lobbyWatcherResetupThread = Executors.newScheduledThreadPool(1);
   private ServerGame m_iGame = null;
   private boolean m_shutDown = false;
-  @SuppressWarnings("deprecation")
-  private final String m_startDate = new Date().toGMTString();
+  private final String m_startDate = new SimpleDateFormat().format(new Date());
```

Comment:
AFAIK this will print a timestamp in the local timezone of the server, and no longer GMT. TBH we should have log4j handle that kind of detail, but for now it looks like we have our own way of doing that..